src/components/Garden/Feed/WrapToken.js

```diff
@@ -19,27 +19,13 @@ import wrappedIcon from '@assets/wrappedIcon.svg'
 import unwrappedIcon from '@assets/unwrappedIcon.svg'
 import claimRewardsIcon from '@assets/rewardsWrapperIcon.svg'
 
-const modeAttributes = {
-  wrap: { icon: unwrappedIcon, button: { mode: 'strong', label: 'Wrap' } },
-  unwrap: {
-    icon: wrappedIcon,
-    button: { mode: 'strong', label: 'Unwrap' },
-    hint:
-      'This amount can be used to vote on proposals. It can be unwrapped at any time.',
-  },
-  claim: {
-    button: { mode: 'normal', label: 'Claim' },
-    icon: claimRewardsIcon,
-  },
-}
-
 function WrapToken({ onClaimRewards, onUnwrapToken, onWrapToken }) {
```

Comment:
let's add this 2 styled components before the `WrapToken` definition, we are going to use them later 
```
const TokenHeaderDiv = styled.div`
  display: flex; 
  align-items: center;
`

const ApySpan = styled.span`
  margin-left: ${0.5 * GU}px; 
`
```